libs/openchallenges/challenge/src/lib/challenge-overview/challenge-overview.component.html

```diff
@@ -26,7 +26,7 @@ <h3 class="section-title">Challenge Details</h3>
         </tr>
         <tr>
           <td class="text-right">Status</td>
-          <td>{{ prettify(challenge.status) }}</td>
+          <td>{{ challenge.status | titlecase }}</td>
```

Comment:
Does this pipe make assumption about the value of the status? I'm asking because this very PR is about fixing an assumption about the casing of challenge input data format that was no longer valid with "MLCube". :-)

Reply:
I get what the pipe does and it's OK given the existing set of status.